.github/workflows/tests.yml

```diff
@@ -160,8 +160,7 @@ jobs:
         uses: ./.github/actions/yarn-nm-install
 
       - name: Install Playwright Browsers
-        # we need 1.37.0 to support Firefox install...
-        run: npx playwright@1.37.0-alpha-jul-20-2023 install --with-deps
+        run: npx playwright@1.38.1 install --with-deps
```

Comment:
I would prefer to keep the version pinned, because otherwise a new release of playwright might break the tests, while they are working locally, which might not be very obvious. Keeping it pinned reverses the problem: in the worst case we are running the CI tests on an older version than locally.